future-apps/payroll/app/src/screens/components/TotalPayroll.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components'
+import TotalPayrollTable from './TotalPayrollTable'
+import { formatCurrency, SECONDS_IN_A_YEAR } from '../../utils/formatting'
+import { totalPaidThisYear, summation, MONTHS_IN_A_YEAR } from '../../utils/calculations'
+
+import { connect } from '../../context/AragonContext'
```

Comment:
This could be changed to `/context/AragonContex` based on https://parceljs.org/module_resolution.html